packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -183,7 +180,7 @@ export default compose( [
 		const insertionPoint = getBlockInsertionPoint();
 		const blockInsertionPointIsVisible = isBlockInsertionPointVisible();
 		const selectedBlock = getSelectedBlock();
-		const isSelectedGroup = selectedBlock && selectedBlock.name === 'core/group';
+		const isSelectedGroup = selectedBlock && ( selectedBlock.name === 'core/group' || selectedBlock.name === 'core/media-text' );
```

Comment:
I thought we were removing all block specific checks like this, not sure why we need them.

Reply:
Replaced with `const hasInnerBlocks = selectedBlock && selectedBlock.innerBlocks.length !== 0;`